packages/dendron-viz/src/dendron.tsx

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env node
 
 import { VaultUtils } from "@dendronhq/common-all";
-import { DendronEngineV2 } from "@dendronhq/engine-server";
 import fs from "fs";
 import React from "react";
 import ReactDOMServer from "react-dom/server";
 import { processDir } from "./process-dendron-notes";
 import { createTree } from "./Tree";
+import { VisualizeCLICommandOpts } from "@dendronhq/dendron-cli";
```

Comment:
You're introducing a circular dependency here: `dendron-viz` imports `dendron-cli` and `dendron-cli` imports `dendron-viz`.

You should write a new type in `dendron-viz` that just lists what `generateSVG` needs. That will be more flexible since you'll probably need to reuse this code in the plugin too.

Reply:
I'll fix it.